src/framework/theme/components/calendar-kit/components/calendar-year-picker/calendar-year-cell.component.ts

```diff
@@ -19,9 +19,12 @@ import { NbCalendarCell } from '../../model';
 
 @Component({
   selector: 'nb-calendar-year-cell',
-  template: `{{ year }}`,
+  template: `
+    <div class="cell-content">
```

Comment:
I already added a comment, but again, do we really need this additional element?

Reply:
Answered here https://github.com/akveo/nebular/pull/2120#discussion_r355343466